specification/recoveryservicesdatareplication/resource-manager/Microsoft.DataReplication/preview/2021-02-16-preview/recoveryservicesdatareplication.json

```diff
@@ -0,0 +1,7002 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Site Recovery Management Service API",
+    "description": "A first party Azure service enabling the data replication.",
+    "version": "2021-02-16-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataReplication/replicationFabrics/{fabricName}/fabricAgents/{draName}": {
+      "get": {
+        "tags": [
+          "Dra"
+        ],
+        "summary": "Gets the Dra.",
+        "description": "Gets the details of the fabric agent.",
+        "operationId": "Dra_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "fabricName",
+            "description": "Fabric name.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9]*$"
+          },
+          {
+            "in": "path",
+            "name": "draName",
+            "description": "Dra name.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9]*$"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns Dra Model on success.",
+            "schema": {
+              "$ref": "#/definitions/DraModel"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ArmError"
```

Comment:
There are definitions of error responses and details in commontypes types.json that you can use instead of declaring your own.
See e.g. `ErrorResponse` and `ErrorDetail` in
https://github.com/Azure/azure-rest-api-specs/blob/be6cd9ccfcb6ba08c1c206627026eabfbff31fc1/specification/common-types/resource-management/v5/types.json

Reply:
Would you please change them manually?